documentation/docs/cli/main_commands.md

```diff
@@ -1769,7 +1768,7 @@ Possible values:  `"chacha20", "aes", "sha3", "shake"` [default: `"aes"`]
 
 `--tag [-t] <TAG>` The tag to associate with the key. To specify multiple tags, use the option multiple times
 
-` <KEY_ID>` The unique id of the key; a random uuid is generated if not specified
+`--key-id <KEY_ID>` The unique id of the key; a random uuid is generated if not specified
```

Comment:
To be consistant, maybe handle this option the same way for other KMIP creation objects ?